public/app/features/integrated-alerting/components/AlertRules/AlertRulesActions/AlertRulesActions.messages.ts

```diff
@@ -1,4 +1,7 @@
 export const Messages = {
   getEnabledMessage: (summary: string) => `Alert rule "${summary}" successfully enabled`,
   getDisabledMessage: (summary: string) => `Alert rule "${summary}" successfully disabled`,
+  createSuccess: 'Alert rule created',
```

Comment:
```suggestion
  createSuccess: 'Alert rule successfully created',
```
QA and PO requested this on other features so let's be consistent.